app/views/remove_document/remove.govspeak.md

```diff
@@ -0,0 +1,5 @@
+If you need to remove this content please [raise a support request and we’ll do it for you](https://support.publishing.service.gov.uk/).
+
+You must include any public explanation or new web address required.
+
+[Full guidance on removing content]( https://www.gov.uk/guidance/content-design/gov-uk-content-retention-and-withdrawal-archiving-policy)
```

Comment:
Looks like you have an extra space character here (and in the retire one)

Reply:
Thanks for spotting :-)